integration-tests/pom.xml

```diff
@@ -273,6 +273,7 @@
                 <module>reactive-messaging-rabbitmq</module>
                 <module>reactive-messaging-rabbitmq-dyn</module>
                 <module>rest-client</module>
+                <module>resteasy-reactive-jackson</module>
```

Comment:
Do we still need this module now that you have added the `ResteasyReactiveUnitTest`?

Reply:
I still struggle to get the MapWithParamConverterTest to work, while the integration test does. 
The way and point in time the parameter converters are found and registered seems to be different.
So maybe it makes sense to keep the module, after all.

Reply:
As another thing to check, you could add a test in https://github.com/quarkusio/quarkus/tree/main/extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test. That should give the same exact timing effects as the module test while keeping the test much "closer"
 to the actual code than the module test.

Reply:
That worked :+1: 
I moved the test there, made the param converter classes inner classes of the test and removed the now-obsolete integration-test module.